src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="page__calendar calendar calendar__day--month-length-29">
+      <div class="calendar__day calendar__day--start-sun"></div>
```

Comment:
```suggestion
    <div class="calendar page__calendar calendar--start-day--sun  calendar--month-length--29">
      <div class="calendar__day"></div>
```
Make it work like that